adapters/ix/ixtest/supplemental/sid.json

```diff
@@ -0,0 +1,184 @@
+{
+  "mockBidRequest": {
+    "id": "test-request-id",
+    "imp": [
+      {
+        "id": "test-imp-id-1",
+        "video": {
+          "mimes": [
+            "video/mp4"
+          ],
+          "minduration": 15,
+          "maxduration": 30,
+          "protocols": [
+            2,
+            3,
+            5,
+            6,
+            7,
+            8
+          ],
+          "w": 940,
+          "h": 560
+        },
+        "ext": {
+          "bidder": {
+            "siteId": "569749",
+            "sid": "1234"
+          }
+        }
+      },
+      {
+        "id": "test-imp-id-2",
+        "video": {
+          "mimes": [
+            "video/mp4"
+          ],
+          "minduration": 15,
+          "maxduration": 30,
+          "protocols": [
+            2,
+            3,
+            5,
+            6,
+            7,
+            8
+          ],
+          "w": 940,
+          "h": 560
+        },
+        "ext": {
+          "bidder": {
+            "siteId": "569750",
+            "sid": "5678"
+          }
+        }
+      }
+    ]
+  },
+  "httpCalls": [
+    {
+      "expectedRequest": {
+        "uri": "http://host/endpoint",
+        "body": {
+          "id": "test-request-id",
+          "imp": [
+            {
+              "id": "test-imp-id-1",
+              "video": {
+                "mimes": [
+                  "video/mp4"
+                ],
+                "minduration": 15,
+                "maxduration": 30,
+                "protocols": [
+                  2,
+                  3,
+                  5,
+                  6,
+                  7,
+                  8
+                ],
+                "w": 940,
+                "h": 560
+              },
+              "ext": {
+                "bidder": {
+                  "siteId": "569749",
+                  "sid": "1234"
+                },
+                "sid": "1234"
```

Comment:
Is it necessary to move `sid` under ext if that value is already present in `ext.bidder.sid`? If your bidder uses `ext.sid` then could you please change it to use `ext.bidder.sid` instead?

Reply:
@gargcreation1992 Yes it is  necessary in the current state to move to ext.sid. We can revisit in the future, but its something we'd like to move for now and get it out